ui/packages/app/src/project_setting/project_info/ProjectInfoForm.js

```diff
@@ -0,0 +1,121 @@
+import React, { useContext, useEffect, useState } from "react";
+import {
+  EuiFlexItem,
+  EuiText,
+  EuiFlexGroup,
+  EuiForm,
+  EuiSpacer,
+  EuiTitle
+} from "@elastic/eui";
+import SubmitProjectInfoForm from "./SubmitProjectInfoForm";
+import config from "../../config";
+import { isDNS1123Label } from "../../validation/validation";
+import { ProjectFormContext } from "../form/context";
+import { Labels } from "../form/Labels";
+import { Stream } from "../form/Stream";
+import { Team } from "../form/Team";
+
+const ProjectInfoForm = ({ originalProject, fetchUpdates }) => {
+  const { project, setStream, setTeam, setLabels } = useContext(
+    ProjectFormContext
+  );
+
+  const [isValidStream, setIsValidStream] = useState(
+    isDNS1123Label(project.stream)
+  );
+  const [isValidTeam, setIsValidTeam] = useState(isDNS1123Label(project.team));
+
+  useEffect(() => {
+    if (!project.team) {
+      setIsValidTeam(false);
+    }
+  }, [project.team]);
+
+  const [isValidLabels, setIsValidLabels] = useState(
+    project.labels.length === 0
+      ? true
+      : project.labels.reduce((labelsValid, label) => {
+          return (
+            labelsValid &&
+            isDNS1123Label(label.key) &&
+            isDNS1123Label(label.value)
+          );
+        }, true)
+  );
+
+  const isDisabled = !config.METADATA_UPDATE_ENABLED;
+
+  return (
+    <>
+      <EuiFlexGroup direction="column">
+        <EuiForm>
+          <EuiFlexItem>
+            <EuiTitle size="s">
+              <h3>Stream</h3>
+            </EuiTitle>
+            <EuiSpacer size="m" />
+            <EuiText size="s" color="subdued">
+              <p>Product stream the project belongs to</p>
+            </EuiText>
+            <EuiSpacer size="m" />
+            <Stream
+              stream={project.stream}
+              setStream={setStream}
+              setIsValidStream={setIsValidStream}
+              isDisabled={isDisabled}
+            />
```

Comment:
Since we are already using the new components `Stream` and `Team`, as well as the updated `Labels` component in `ProjectInfoForm.js`, can we also update `ProjectForm.js` to use these new/updated components?